pkg/network/setup/netstat_test.go

```diff
@@ -141,7 +141,7 @@ var _ = Describe("netstat", func() {
 			Expect(setup.NetStat.PodInterfaceVolatileDataIsCached(setup.Vmi, primaryNetworkName)).To(BeTrue())
 		})
 
-		It("run status and expect 2 interfaces to be reported based on guest-agent data", func() {
+		It("run status and expect 2 interfaces to be reported based on pod and guest-agent data", func() {
```

Comment:
Please add another scenario - the pod has no IP address, the IP is reported from the GA only.

Reply:
Done